### PR TITLE
Rename `apollo` to `ap`

### PR DESCRIPTION
**Pull request by jbaxleyiii**
This will reduce the migration burden from existing users and is shorter to type

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -155,9 +155,9 @@ jobs:
         shell: bash
         run: |
           touch sha256.txt
-          echo "Linux: $(sha256sum -b ./linux/apollo | cut -d ' ' -f1)" >> sha256.txt
-          echo "Windows: $(sha256sum -b ./windows/apollo.exe | cut -d ' ' -f1)" >> sha256.txt
-          echo "MacOS: $(sha256sum -b ./macos/apollo | cut -d ' ' -f1)" >> sha256.txt
+          echo "Linux: $(sha256sum -b ./linux/ap | cut -d ' ' -f1)" >> sha256.txt
```
Comment (queerviolet) [inline .github/workflows/build.yml:158]: should we use `${{ RELEASE_BIN }}` here?

File: cli/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
-name = "apollo"
-version = "0.0.1"
-authors = ["jbaxleyiii <james@apollographql.com>", "ashi <ashi@apollographql.com>"]
+name = "ap"
```
Comment (queerviolet) [inline cli/Cargo.toml:2]: thinking about this, there's no reason we have to rename it here, right? We could name this `cli` or `apollo` or `apollo-cli`?

Comment (queerviolet) [reply cli/Cargo.toml:2]: so, here's the thing… it seems like for the installed binary, we maybe want to go with `ap` but also sortof want to keep `apollo` if we can, and i'm still holding out a little bit of hope for `a`/`an`. in any case, i think it's safe to say that the final installed command name isn't set in stone.

So maybe the crate shouldn't be named the final installed binary, but instead `apollo-cli` or something that will definitely not be the final name of the installed command, but is the name of the crate.

This way, we don't have to change the tests and scripts as we iterate on command names. You'll always do `cargo run --bin=apollo-cli`, the tests can always call `Command::cargo_bin("apollo-cli")` and so on

Comment (jbaxleyiii) [reply cli/Cargo.toml:2]: @queerviolet ah I see what you are getting at now. Sure thing! I'll make the adjustments. Do you prefer `apollo-cli` or just `cli`?

File: cdn/src/handler.ts [hunk 3]
```diff
@@ -62,6 +63,20 @@ export async function handleRequest(event: FetchEvent): Promise<Response> {
...
+
+        if (!product) {
+          const headers = {
+            accept: event.request.headers.get("accept") || "",
+            ["accept-encoding"]: event.request.headers.get("accept-encoding") || "",
+            ["accept-charset"]: event.request.headers.get("accept-charset") || "",
+            ["accept-language"]: event.request.headers.get("accept-language") || "",
+          }
+          const negotiator = new Negotiator({ headers });
+          if (!negotiator.mediaTypes().includes("text/html")) {
```
[the remark this answers is not in the document]
Comment (glasser) [reply cdn/src/handler.ts:75]: drive-by note: I love the idea of a single file that is both HTML and shell script, but it's also pretty nice to be able to actually view the install script in a web browser, so preserving that somehow would be nice :)

[the remark this answers is not in the document]
Comment (jbaxleyiii) [reply cdn/src/handler.ts:75]: @queerviolet we certainly could do something like that akin to what [Meteor](https://install.meteor.com/) does in their install script. I __personally__ really like having the designed landing page if people got here via html and don't think it adds too much complexity to the worker.

[the remark this answers is not in the document]
Comment (jbaxleyiii) [reply cdn/src/handler.ts:75]: @glasser so you would prefer the shell script in the browser vs the custom index.html page that tells you how to install it?

[the remark this answers is not in the document]
Comment (queerviolet) [reply cdn/src/handler.ts:75]: What if we just included bits of HTML but didn't use the HTML to block viewing the code?

[the remark this answers is not in the document]
Comment (queerviolet) [reply cdn/src/handler.ts:75]: I actually thought about doing that but wanted to preserve the look and feel entirely, but there's no reason we gotta. Just a sec, I'll whip something up ;)

**Pull request merged**